packages/app-project/src/helpers/fetchWorkflowsHelper/fetchWorkflowsHelper.js

```diff
@@ -40,28 +49,39 @@ async function fetchPreviewImage (subjectSet, env) {
   subjectSet.subjects = linked.subjects
 }
 
+async function buildWorkflow(workflow, displayName, subjectSets, isDefault) {
+  const subjectSetCounts = await fetchWorkflowCellectStatus(workflow)
+  const workflowSubjectSets = workflow.links.subject_sets
+    .map(subjectSetID => {
+      const subjectSet = subjectSets.find(subjectSet => subjectSet.id === subjectSetID)
+      subjectSet.availableSubjects = subjectSetCounts[subjectSetID]
```

Comment:
Subject sets sent back from `fetchWorkflowData` are all subject sets from all workflows, so should include every linked set for each individual workflow.

It would be useful to test this for a realistic data set. The test project only has two workflows and two sets. What happens when you want to link 100 volumes of transcription material to your workflow?

Reply:
It turns out this is undefined for AnnoTate (but then, so are a bunch of other values that we would expect to exist, like workflow display name.)
http://local.zooniverse.org:3000/projects/drrogg/annotate?env=production

I suspect what's going on for AnnoTate is that completed sets aren't present in the Cellect response.

I'm now playing with the code to see what needs to be hacked to make AnnoTate work.
![Screenshot of the subject picker showing completed subject sets for AnnoTate.](https://user-images.githubusercontent.com/59547/108538465-baf33180-72d6-11eb-8f80-faf8d249194a.png)

Reply:
AnnoTate's workflow has 1198 linked subject sets, returned in pages of 20. I've completely rewritten the data fetching code to start dealing with that. There's a separate API request for the subject set data, using the subject IDs that are returned by Cellect.